ctru-rs/src/services/cam.rs

```diff
@@ -0,0 +1,1027 @@
+use bitflags::bitflags;
+use ctru_sys::Handle;
+
+pub struct Cam(());
+
+bitflags! {
+    #[derive(Default)]
+    pub struct CamPort: u32 {
+        const PORT_NONE = ctru_sys::PORT_NONE;
+        const PORT_CAM1 = ctru_sys::PORT_CAM1;
+        const PORT_CAM2 = ctru_sys::PORT_CAM2;
+        const PORT_BOTH = ctru_sys::PORT_BOTH;
+    }
+}
+
+bitflags! {
+    #[derive(Default)]
+    pub struct CamSelect: u32 {
+        const SELECT_NONE      = ctru_sys::SELECT_NONE;
```

Comment:
Since the enum is named CamSelect, you can remove the `SELECT_` prefix from all variants.